src/app/workflow/dag/dag.component.ts

```diff
@@ -190,7 +203,6 @@ export class DagComponent implements OnInit, AfterViewChecked {
   download() {
     if (this.cy) {
       const pngDAG = this.cy.png({ full: true, scale: 2 });
-      const name = this.workflow.repository + '_' + this._selectedVersion.name + '.png';
```

Comment:
Not sure what happened here.

Reply:
Good catch, thanks! At one point I had all my changes in dag.component.ts, before I decided to pull them into their own component, and I must have left this by accident.